flink-table/flink-table-planner-blink/src/test/resources/org/apache/flink/table/planner/plan/stream/sql/DagOptimizationTest.xml

```diff
@@ -959,15 +951,15 @@ LogicalSink(name=[`default_catalog`.`default_database`.`upsertSink`], fields=[b,
     </Resource>
     <Resource name="planAfter">
       <![CDATA[
-Sink(name=[`default_catalog`.`default_database`.`upsertSink`], fields=[b, c, a_sum], updateAsRetraction=[false], accMode=[Acc])
-+- Union(all=[true], union=[b, c, a_sum], updateAsRetraction=[false], accMode=[Acc])
-   :- GroupAggregate(groupBy=[b, c], select=[b, c, SUM(a) AS a_sum], updateAsRetraction=[false], accMode=[Acc])
-   :  +- Exchange(distribution=[hash[b, c]], updateAsRetraction=[true], accMode=[Acc])
-   :     +- TableSourceScan(table=[[default_catalog, default_database, MyTable, source: [TestTableSource(a, b, c)]]], fields=[a, b, c], updateAsRetraction=[true], accMode=[Acc], reuse_id=[1])
-   +- Calc(select=[CAST(1:BIGINT) AS b, c, a_sum], updateAsRetraction=[false], accMode=[Acc])
-      +- GroupAggregate(groupBy=[c], select=[c, SUM(a) AS a_sum], updateAsRetraction=[false], accMode=[Acc])
-         +- Exchange(distribution=[hash[c]], updateAsRetraction=[true], accMode=[Acc])
-            +- Calc(select=[c, a], updateAsRetraction=[true], accMode=[Acc])
+Sink(name=[`default_catalog`.`default_database`.`upsertSink`], fields=[b, c, a_sum], changelogMode=[NONE])
```

Comment:
Since FLIP-95 new sink interfaces, PK of sink is defined in DDL, not derived from query anymore. Deriving primary key from query confused many users and make Flink SQL not flexible. So I will add the PK information once we integrate with new sink interface.